sqlglot/generator.py

```diff
@@ -1335,14 +1335,15 @@ def sessionparameter_sql(self, expression: exp.SessionParameter) -> str:
     def placeholder_sql(self, expression: exp.Placeholder) -> str:
         return f":{expression.name}" if expression.name else "?"
 
-    def subquery_sql(self, expression: exp.Subquery) -> str:
+    def subquery_sql(self, expression: exp.Subquery, sep: str = " AS ") -> str:
```

Comment:
you shouldn't have different signatures in subclass overrides like this

Reply:
We should probably change `alias_sql` too then. I assume that Mike tried to replicate that logic.